src/js/Calculator.js

```diff
@@ -0,0 +1,82 @@
+import {
+  OPERATORS,
+  MESSAGE,
+  MAXIMUM_DIGITS_LENGTH,
+} from "./utils/constants.js";
+
+export default class Calculator {
+  constructor(displayElement) {
+    this.operator = "";
+    this.tmpValue = "";
+    this.displayElement = displayElement;
```

Comment:
```suggestion
    this.displayValue = displayElement.innerText;
```
와 같이 innerText 값을 직접 가져오는건 어떨까요? 이후의 쓰임을 보았을때 dom element 자체의 참조가 필요 없어보여요!

Reply:
이제와서 생각해보면 dom elemet 가 아닌 수지님 말씀과 같이 받았으면 더 좋았을 것 같네요.

세심한 리뷰 감사합니다 :)